shared/common-adapters/multi-line-input.desktop.js

```diff
@@ -54,7 +60,8 @@ export default class MultiLineInput extends Component<void, Props, State> {
 
 const containerStyle = {
   ...globalStyles.flexBoxColumn,
-  position: 'relative'
+  position: 'relative',
+  justifyContent: 'flex-end'
```

Comment:
there was a bug that the underline would be placed from top to bottom, but text box was always placed on bottom. This fixes it to justify at the bottom.